src/Client.cpp

```diff
@@ -282,6 +282,7 @@ bool Client::OnBeforeBrowse(
     CefRefPtr<CefBrowser> browser,
     CefRefPtr<CefFrame> frame,
     CefRefPtr<CefRequest> request,
+	bool user_gesture,
```

Comment:
As below. Bad indent. Don't use tabs!

Reply:
ok, should I create a new pull request? :)

Reply:
No. Just add new commit or amend last commit

Reply:
done